contrib/ckan/Dockerfile

```diff
@@ -11,15 +10,18 @@ ENV CKAN_SITE_URL http://localhost:5000
 
 # Install required packages
 RUN apt-get -q -y update && apt-get -q -y upgrade && DEBIAN_FRONTEND=noninteractive apt-get -q -y install \
-		python-dev \
+        libssl-dev \
+        libffi-dev \
+        build-essential \
+        python-dev \
         python-pip \
         python-virtualenv \
         libpq-dev \
         git-core \
-	&& apt-get -q clean
+    && apt-get -q clean
 
-# Download CKAN
-RUN git clone https://github.com/ckan/ckan.git /ckan
+# Download CKAN		
+ RUN git clone https://github.com/ckan/ckan.git /ckan		
```

Comment:
Any reason for the space/tabs before and on these lines?

Reply:
Notes. Will edit that